crates/gpui/src/platform/mac/window.rs

```diff
@@ -1248,7 +1251,8 @@ extern "C" fn handle_key_event(this: &Object, native_event: id, key_equivalent:
         let is_composing =
             with_input_handler(this, |input_handler| input_handler.marked_text_range())
                 .flatten()
-                .is_some();
```

Comment:
`is_composing` will be `false` when deleting the last char when IME composing, then `KeyDown("backspace")` is propagated, leads to mis-deleting other char.